src/coreclr/jit/codegencommon.cpp

```diff
@@ -5210,7 +5210,7 @@ void CodeGen::genReserveEpilog(BasicBlock* block)
 
     assert(block != nullptr);
     const VARSET_TP& gcrefVarsArg(GetEmitter()->emitThisGCrefVars);
-    bool             last = (block->bbNext == nullptr);
+    bool             last = (block->IsLast());
```

Comment:
```suggestion
    bool             last = block->IsLast();
```
or just substitute it in the call below and eliminate the `last` var